src/components/OptionsSelector.js

```diff
@@ -226,7 +226,7 @@ class OptionsSelector extends Component {
             focusedIndex: newFocusedIndex,
         });
 
-        if (newOptions.length <= newFocusedIndex) {
+        if (newOptions.length < newFocusedIndex) {
```

Comment:
@roryabraham when the search query has no results, this check will fail.
So we'll scroll to index 0 which doesn't exist, causing the app to crash.

Steps
1. ⌘ + K
2. Search for an user which doesn't exist.

Result: 
<img width="509" alt="image" src="https://user-images.githubusercontent.com/29673073/177022417-65fe5877-eeda-4b34-8560-fc8ea5e22a5e.png">

Reply:
Also causing issue https://github.com/Expensify/App/issues/9685

Reply:
Thanks for pointing this out. Revert PR here: https://github.com/Expensify/App/pull/9690